daprdocs/content/en/developing-applications/building-blocks/actors/actors-overview.md

```diff
@@ -26,6 +26,17 @@ Every actor is defined as an instance of an actor type, identical to the way an
 
 <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/0y7ne6teHT4?si=dWNgtsp61f3Sjq0n&amp;start=10797" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
+## Actor types and IDs
```

Comment:
I am checking where is this coming from.. I didn't made these changes.. but they popped up when I rebased.